src/database/migrations/20220417175812-modify_users_add_new_fields.js

```diff
@@ -0,0 +1,78 @@
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    return Promise.all([
+      queryInterface.addColumn('Users', 'gender', {
+        type: Sequelize.ENUM,
+        values: ['male', 'female', 'prefer not say'],
+        allowNull: true,
+      }),
+      queryInterface.addColumn('Users', 'dob', {
+        type: Sequelize.DATEONLY,
+        allowNull: true,
+      }),
+      queryInterface.addColumn('Users', 'nationality', {
+        type: Sequelize.STRING,
+        defaultValue: 'rwandan',
+        allowNull: true,
+      }),
+      queryInterface.addColumn('Users', 'department', {
+        type: Sequelize.STRING,
+        allowNull: true,
+      }),
+      queryInterface.addColumn('Users', 'preferredLanguage', {
+        type: Sequelize.STRING,
+      }),
+      queryInterface.addColumn('Users', 'preferredCurrency', {
+        type: Sequelize.STRING,
+        allowNull: true,
+      }),
+      queryInterface.addColumn('Users', 'address', {
+        type: Sequelize.STRING,
+        allowNull: true,
+      }),
+      queryInterface.addColumn('Users', 'lineManager', {
+        type: Sequelize.STRING,
```

Comment:
isn't this supposed to be integer type? i think it has to be id of line manager

Reply:
No, I haven't been updated to what the line Manager should hold.